### PR TITLE
Language corrections, changes for more clarity, simplify some phrases

### PR DESCRIPTION
**Pull request by sypets**

### CODE REVIEW

File: Documentation/6-Persistence/Index.rst [hunk 1]
```diff
@@ -3,15 +3,15 @@
 Setting up the persistence layer
 ================================
 
-As already insinuated in previous chapters it is the Persistence Layer which
-takes care about conserving Domain Objects durably. Thus, in this chapter we
-will discuss the steps being important for doing that.
+As already mentioned in previous chapters it is the Persistence Layer which
```
Comment (tomwarwick) [inline Documentation/6-Persistence/Index.rst]: As discussed in previous chapters, it is the Persistence Layer which
	takes care of conserving Domain Objects. In this chapter we
	will describe the necessary steps for doing that.

File: Documentation/6-Persistence/Index.rst [hunk 2]
```diff
@@ -3,15 +3,15 @@
...
+takes care of conserving Domain Objects durably. In this chapter we
+will describe the necessary steps for doing that.
 
 It is important to get a grasp of the lifecycle of a Domain Object to fully
 understand the Persistence Layer. When instantiating a Domain Object we essentially
-put its data into a certain sector of the Main Memory. At that time, it is in
+store it in RAM. At that time, it is in
```
Comment (tomwarwick) [inline Documentation/6-Persistence/Index.rst]: store it in memory (just my pesonal preference)

File: Documentation/6-Persistence/Index.rst [hunk 3]
```diff
@@ -3,15 +3,15 @@
...
 a transient, i.e. volatile, state. When TYPO3 delivered the rendered website
-the sector of the Main Memory is freed by PHP and may be overwritten with
+the sector of the RAM is freed by PHP and may be overwritten with
```
Comment (tomwarwick) [inline Documentation/6-Persistence/Index.rst]: piece of memory

Comment (tomwarwick) [reply Documentation/6-Persistence/Index.rst]: freed up by

File: Documentation/6-Persistence/Index.rst [hunk 4]
```diff
@@ -21,27 +21,27 @@ other data. Thereby the saved data will be lost together with the Domain Object.
 If Domain Objects should be available within several page loads they have to be
 transferred into a persistent state. This is being done in Extbase by putting
 the Domain objects into a Repository. When the script is finished doing its work,
-the Repository takes care about saving the volatile data in a durable saving state.
-Normally this is the database which is used by TYPO3 but may, for example,
-also be a text file.
+the Repository takes care of saving the data into a durable storage.
```
Comment (tomwarwick) [inline Documentation/6-Persistence/Index.rst]: part of the repositories role is to ensure that the integiry of the data is maintained at all times.
Typically, this is the database which is being used by TYPO3.

Comment (sypets) [reply Documentation/6-Persistence/Index.rst]: @tomwarwick Thank you! I will use your suggestions.

**Pull request merged**